pkg/pillar/agentlog/http-debug_test.go

```diff
@@ -89,19 +90,20 @@ func TestListenDebug(t *testing.T) {
 		t.Skip()
 	}
 
-	err = os.MkdirAll("/hostfs/etc/", 0755)
+	err = os.MkdirAll(types.HostFSEtcDir, 0755)
```

Comment:
This is a test file hence it doesn't run on EVE but in the build environment, hence I'm confused by the references to /hostfs and /persist which only exist in the EVE runtime environment.